config/prov/deploy_config.yaml

```diff
@@ -63,6 +63,15 @@ nodes_3:
     dix_spare : 0
     cvg_per_node : 3
     data_disk_per_cvg : 1
+  config_6:
```

Comment:
We have a similar config defined in config_2. 
Can you please explain why this is needed. 
cc @pragamj